lib/r10k/git/errors.rb

```diff
@@ -10,24 +10,24 @@ class UnresolvableRefError < GitError
       attr_reader :ref
       attr_reader :git_dir
 
-      def initialize(*args)
+      def initialize(mesg, options = {})
+        if mesg.is_a? Hash
```

Comment:
I will note that this type of variable-arity pandering (!) doesn't seem to exist in any of the other r10k errors.